src/neurosnap/protein.py

```diff
@@ -1014,53 +1011,41 @@ def foldseek_search(
+  output_format: Optional[str] = None,
+  output_path: Optional[str] = None,
+  return_df: bool = True,
+) -> Optional[pd.DataFrame]:
   """Submits a BLAST job to the EBI NCBI BLAST web service, checks the status periodically, and retrieves the result.
   The result can be saved either as an XML or FASTA file. Optionally, a DataFrame with alignment details can be returned.
 
   Parameters:
-    sequence : str or Protein
-      The input amino acid sequence as a string or a Protein object. If a Protein object is provided with multiple chains,
-      an error will be raised, and the user will be prompted to provide a single chain sequence using the `get_aas` method.
-    email : str
-      The email address to use for communication if there is a problem.
-    matrix : str, optional
-      The scoring matrix to use (default is "BLOSUM62"). Must be one of ["BLOSUM45", "BLOSUM62", "BLOSUM80", "PAM30", "PAM70"].
-    alignments : int, optional
-      The number of alignments to display in the result (default is 250). the number alignments must be
-    scores : int, optional
-      The number of scores to display in the result (default is 250).
-    evalue : float, optional
-      The E  threshold for alignments (default is 10). must be one of the following: 50, 100, 250, 500, 750, 1000.
-    filter : bool, optional
-      Whether to filter low complexity regions (default is False).
-    gapalign : bool, optional
-      Whether to allow gap alignments (default is True).
-    database : str, optional
-      The database to search in (default is "uniprotkb_refprotswissprot"). Must be one of:
-      ["uniprotkb_refprotswissprot", "uniprotkb_pdb", "uniprotkb", "afdb", "uniprotkb_reference_proteomes",
-      "uniprotkb_swissprot", "uniref100", "uniref90", "uniref50", "uniparc"].
-    output_format : str, optional
-      The format in which to save the result. Either "xml" or "fasta". If None, no file will be saved (default is None).
-    output_path : str, optional
-      The file path to save the output. This is required if `output_format` is specified.
-    return_df : bool, optional
-      Whether to return a DataFrame with alignment details (default is True).
+    sequence: The input amino acid sequence as a string or a Protein object.
+      If a Protein object is provided with multiple chains, an error will be raised, and the user will be prompted to provide a single chain sequence using the `get_aas` method.
+    email: The email address to use for communication if there is a problem.
+    matrix: The scoring matrix to use (default is "BLOSUM62"). Must be one of ["BLOSUM45", "BLOSUM62", "BLOSUM80", "PAM30", "PAM70"].
+    alignments: The number of alignments to display in the result (default is 250). the number alignments must be
+    scores: The number of scores to display in the result (default is 250).
+    evalue: The E  threshold for alignments (default is 10). must be one of the following: 50, 100, 250, 500, 750, 1000.
```

Comment:
evalue: The E threshold for alignments (default is 10). must be one of the following: 0.00001, 0.0001, 0.001, 0.01, 0.1, 1.0, 10, 100, 1000.